docs/develop/configuration.md

```diff
@@ -187,96 +122,99 @@ requirements:
 - scipy==1.7.3
 ```
 
-## system_packages
+### system_packages
 
-Truss assumes the debian operating system. This field can be used to specify any
+Truss assumes the Debian operating system. This field can be used to specify any
 system packages that you would typically install using `apt`.
 
-e.g.
-```config.yaml
+```yaml
 system_packages:
 - ffmpeg
 - libsm6
 - libxext6
 ```
 
-## environment_variables
+### environment_variables
+
+{% hint style="danger" %}
+Do not store secret values directly in environment variables (or anywhere in the config file). See the section on secrets below for information in properly managing secrets.
```

Comment:
Not sure but should it be `information on` instead of `information in`. That's where my command of english breaks down.

Reply:
You are absolutely right, that was a typo on my part